volsync.yaml

```diff
@@ -0,0 +1,72 @@
+package:
+  name: volsync
+  version: "0.13.0"
+  epoch: 0
+  description: Asynchronous data replication for Kubernetes volumes
+  copyright:
+    - license: AGPL-3.0-or-later
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/backube/volsync
+      tag: v${{package.version}}
+      expected-commit: 34b8e30c6354bb8380dc77310ae07ba8687b3e5b
+
+  - uses: go/bump
+    with:
+      deps: |-
+        github.com/go-viper/mapstructure/v2@v2.3.0
+
+  - uses: go/build
+    with:
+      ldflags: -X=main.volsyncVersion=v${{package.version}}
+      packages: ./cmd
+      output: manager
+
+subpackages:
+  - name: ${{package.name}}-cli
+    description: CLI for volsync
+    pipeline:
+      - uses: go/build
+        with:
+          packages: ./kubectl-volsync
+          output: kubectl-volsync
+    test:
+      pipeline:
+        - name: "basic test"
+          runs: |
+            kubectl-volsync --version
+            kubectl-volsync --version
+
+update:
+  enabled: true
+  github:
+    identifier: backube/volsync
+    strip-prefix: v
```

Comment:
We need to add `use-tag: true` in this case and add `-rc`, `helm-`, etc to ignore-regex-patterns since they do maintain tags for release candidates.
- https://github.com/backube/volsync/tags